src/nvim/syntax.h

```diff
@@ -3,6 +3,7 @@
 
 #include <stdbool.h>
 
+#include "nvim/globals.h"
```

Comment:
seems like this isn't needed so I removed it in the merge-commit.

Reply:
It didn't work last time when I used TriState for `boldp` in patch 8.0.0831 (
0c0318f)